README.md

```diff
@@ -252,3 +252,16 @@ oauth:
 ## Credits
 
 - Free email provider domains list provided by: https://gist.github.com/tbrianjones/5992856/
+
+## Create release tag before deployment
+
+```sh
+# Create release tag when deploying to staging or production
+# For staging, prefix the tag with `staging-`. e.g. staging-20210506.0
```

Comment:
The date format in the example is incorrect. There should be hyphens between date components.

Reply:
Updated, thanks!!